include/OpenSpaceToolkit/Astrodynamics/EventCondition/AngularCondition.hpp

```diff
@@ -0,0 +1,154 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_EventCondition_AngularCondition__
+#define __OpenSpaceToolkit_Astrodynamics_EventCondition_AngularCondition__
+
+#include <OpenSpaceToolkit/Core/Containers/Pair.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/Physics/Units/Derived/Angle.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/EventCondition.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace eventcondition
+{
+
+using ostk::core::types::Real;
+using ostk::core::types::String;
+using ostk::core::ctnr::Pair;
+
+using ostk::physics::units::Angle;
+
+using ostk::astro::trajectory::State;
+
+/// @brief                      An Event Condition that deals with angles.
```

Comment:
```suggestion
/// @brief                      An Event Condition that deals with angles. Automatically handles wrapping angle to [0,360).
```

Something like that? Just to explain why it's useful to have this separate. The problem that caused this isn't super obvious. We all missed it initially 😉